torchdynamo/eval_frame.py

```diff
@@ -104,15 +104,16 @@ def __init__(
         patch_fn()
 
     def __enter__(self):
-        log.warning(
-            (
-                "Using TorchDynamo with a context manager will be deprecated soon."
-                "Please read https://github.com/pytorch/torchdynamo#usage-example "
-                "to use TorchDynamo using an annotation."
-            )
-        )
         if config.raise_on_ctx_manager_usage:
             raise RuntimeError("TorchDynamo is used with a context manager")
+        else:
```

Comment:
Should we just remove the warning now that raising is the default behavior?  If someone sets the config they should know what they are doing and the warning will just be annoying

Reply:
Yeah, thats correct. Will remove it.